### PR TITLE
Release 0.4.4-beta

### PR DESCRIPTION
**Pull request by tsterbak**
This Release adds:

- Samsung A5 2017 (a5y17lte) thanks to @Gredin67
- Support for device code `beyond2` for Samsung Galaxy S10+.
- Support for device code `evert_n` for Motorola Moto G6 plus.
- Add config file for Samsung Tab S2 gts210vewifi T813 thanks to @freechelmi 
- a "request support for device" button if no config for the device code is found

Fixes:
- Update the unlocking instructions for FP3/4 and add
fastboot_unlock_critical command.

Changes:
- Set a min width/height for the window to solve basic responsiveness issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ Vendor | Device Name | CodeName | Models | Status
 Samsung | Galaxy J7 2015 | j7elte | | tested
 Samsung | Galaxy A3 2017 | a3y17lte | SM-A320FL | tested
 Samsung | Galaxy A5 2016 | [a5xelte](https://wiki.lineageos.org/devices/a5xelte) | SM-A510F | tested
+Samsung | Galaxy A5 2017 | [a5y17lte](https://doc.e.foundation/devices/a5y17lte) | | tested
```
Comment (MagicLike) [inline README.md:79]: Wouldn't it be better to link the LineageOS wiki to stay consistent?

Comment (tsterbak) [reply README.md:79]: Fair point! :+1:

Comment (tsterbak) [inline README.md:79]:
```suggestion
Samsung | Galaxy A5 2017 | [a5y17lte](https://wiki.lineageos.org/devices/a5y17lte) | | tested
```

**Comment by Gredin67** [comment]
Note that I did not test with LineageOS (but I could)

**Comment by MagicLike** [comment]
> Note that I did not test with LineageOS (but I could)

Would be newt if you could do it!

**Comment by freechelmi** [comment]
Hello , anything blocking us to release this ?

**Comment by MagicLike** [comment]
> Hello , anything blocking us to release this ? 

I guess the issues on the 0.4.4 milestone...

**Comment by tsterbak** [comment]
This is a bit of a question of a release strategy and I'm not sure what a good approach is.

We could do a minor release now under 0.4.4 and then work on a larger release with more new features and fixes under 0.5.0.

Right now, I was settling for less but larger releases but we could also do otherwise.
What are your opinions on this @freechelmi @MagicLike ?

**Comment by MagicLike** [comment]
> This is a bit of a question of a release strategy and I'm not sure what a good approach is.
> 
> We could do a minor release now under 0.4.4 and then work on a larger release with more new features and fixes under 0.5.0.
> 
> Right now, I was settling for less but larger releases but we could also do otherwise. What are your opinions on this @freechelmi @MagicLike ?

I think it would be better to release 0.4.4 now, as it contains new devices, expands support for existing devices, fixes the scaling issue and even makes it easier to use for newcomers, who did not check if their device is supported. - Other stuff that can't be fixed quickly should be moved to 0.5 or even a possible  0.4.5...

**Pull request merged**